### PR TITLE
add new entries and translations

### PR DESCRIPTION
**Pull request by stardustichi**

### CODE REVIEW

File: src/data/skill-name.csv [hunk 1]
```diff
@@ -50,7 +50,9 @@
 "きっとその先へ…！","Surely, beyond...!"
 "Pride of KING","Pride of KING"
 "絶対は、ボクだ","It's going to be me"
+"ゲインヒール・スペリアー","Gain Superior Heal"
 "最強の名を懸けて","For the Strongest Name"
+"コンドル猛撃波","Condor Raging Attack"
```
Comment (robflop) [inline src/data/skill-name.csv:55]: This drops out the last word 波, "Wave". If being more eloquent, this is probably something like "Wave of Furious Condor Attacks", but we need to shorten that down a bit.

File: src/data/support-title.csv [hunk 2]
```diff
@@ -67,5 +67,8 @@
 "[召しませふぁーすとBite！]","[Try First with Bite!]"
 "[おもい、ねがう]","[A Feeling, a Wish]"
 "[Head-on fight！]","[Head-on fight!]"
+"[明日は全国的に赤でしょう♪]","[Crimson to the Entire Country Tomorrow♪]"
```
Comment (robflop) [inline src/data/support-title.csv:70]: I feel like this should be more like "Tomorrow the entire country will be dyed red" (crimson works too ig), but that would need to be shortened down of course.

File: src/data/support-title.csv [hunk 3]
```diff
@@ -67,5 +67,8 @@
...
+"[今宵、円舞曲にのせて]","[Tonight, Let's Dance to the Waltz]"
+"[ウマ王伝説・最強になった件]","[Legend of the Uma King・The Strongest]"
```
Comment (robflop) [inline src/data/support-title.csv:72]: The first part here is fine, but the 2nd completely leaves out most of the content. It reads more like "How [he] became the Strongest". That's too long though, so we'd need to reword it to be better, while still including relevant content.

File: src/data/trainer-title.csv [hunk 4]
```diff
@@ -66,6 +66,8 @@
 "お金持ち","Big-time Rich"
 "ストーリーテラー","Storyteller"
 "駆け出しの一等星","Fledgling First-Magnitude Star"
+"小さながんばり屋","A Little Hard Worker"
+"夢を掴む特急券","Limited Ticket to Seize a Dream"
```
Comment (robflop) [inline src/data/trainer-title.csv:70]: I'd modify this to be "Special Ticket" instead of limited. In addition to that, we could maybe go with something like "Dream-seizing Special Ticket" if we want to keep it a bit more concise.

File: src/data/skill-desc.csv [hunk 5]
```diff
@@ -412,6 +414,8 @@
 "レース終盤で3番手の時に負けそうに\nなると速度がわずかに上がる","<size=22>When you're in 3rd place at the end of the race and about to lose, your speed slightly increases</size>"
 "レースを冷静に運ぶことができると残り\n200mで実力発揮、速度が少し上がる","<size=22>After proceeding through the race calmly, your speed increases a little during the last 200m</size>"
 "最終直線で前の方で追いすがっていると\n速度がわずかに上がる","<size=22>Slightly increase speed in the last straight when chasing someone close to the lead</size>"
+"中盤に後ろの方で順位を下げた時に\n持久力をわずかに回復する","<size=22>During Middle Leg, recover slight stamina when losing position in the back of the pack</size>"
```
Comment (robflop) [inline src/data/skill-desc.csv:417]: "recover slight stamina" -> "recover stamina slightly" or "slightly recover stamina"

File: src/data/skill-desc.csv [hunk 6]
```diff
@@ -412,6 +414,8 @@
...
 "最終コーナー以降に前めで力が尽きかけ\n詰め寄られると速度がわずかに上がる","<size=22>Slightly increase speed after the last corner when stamina runs out being one of the lead</size>"
+"最終コーナーで中団から前を狙う時\n加速力をわずかに上げる","<size=22>On last corner, gain slight acceleration when aiming for the lead from the middle pack</size>"
```
Comment (robflop) [inline src/data/skill-desc.csv:419]: "gain slight acceleration" -> "gain acceleration slightly" or "slightly gain acceleration"

**Review by robflop — CHANGES_REQUESTED** [review]

File: src/data/skill-name.csv (hunk 1 above)
Comment (FabulousCupcake) [reply src/data/skill-name.csv:55]: Condor Onslaught?

File: src/data/trainer-title.csv (hunk 4 above)
Comment (stardustichi) [reply src/data/trainer-title.csv:70]: sounds good to me

File: src/data/support-title.csv (hunk 3 above)
Comment (stardustichi) [reply src/data/support-title.csv:72]: i really cant think of the proper words for the second part, so if anyone has a better one please let me know

File: src/data/skill-name.csv (hunk 1 above)
Comment (stardustichi) [reply src/data/skill-name.csv:55]: wave of condor onslaughts?

File: src/data/skill-desc.csv (hunk 5 above)
Comment (gertasik) [reply src/data/skill-desc.csv:417]: Also I'm pretty sure other change_order_onetime>0 skills use "when overtaken", not "when losing position"

File: src/data/skill-name.csv (hunk 1 above)
Comment (gertasik) [reply src/data/skill-name.csv:55]: Condor Onslaught sgtm

File: src/data/skill-desc.csv (hunk 5 above)
Comment (stardustichi) [reply src/data/skill-desc.csv:417]: i guess i can change that, they pretty much the same, i was just trying to match the japanese one

File: src/data/skill-name.csv (hunk 1 above)
Comment (robflop) [reply src/data/skill-name.csv]: Good with that too

File: src/data/support-title.csv (hunk 3 above)
Comment (gertasik) [reply src/data/support-title.csv:72]: Well it does sound like a name for the first entry in a line of books, but I don't think we need to go for word-for-word, since it's a one-time title name.

I do agree that it sounds a bit stiff tho, something like "The Legend of the Strongest Uma King" would probably roll off the tongue better.

Comment (robflop) [reply src/data/support-title.csv:72]: Staying literal probably won't cut it yea, what you suggested sounds better to me already.

Comment (gertasik) [reply src/data/support-title.csv:72]: Or, if you really wanted to include the second part and keep it really flowery, "The Legend of the Strongest Uma King and How He Came to Be", but that's getting a bit long lol

Comment (robflop) [reply src/data/support-title.csv:72]: That's definitely not doable for the patch i think lol

Comment (stardustichi) [reply src/data/support-title.csv]: yeah i think ill go with what gertasik suggests here, thanks!

File: src/data/trainer-title.csv [hunk 7]
```diff
@@ -205,6 +207,7 @@
 "キミの夢へと走り出せ！","Chase Your Dreams"
 "Brand-new Friend","Brand-new Friend"
 "花咲く乙女のJunePride","JunePride of the Blooming Maidens"
+"幻想世界ウマネスト","Fantasy World Umanest"
```
Comment (robflop) [inline src/data/trainer-title.csv:210]: "Uma Nest" would be better here I think, since this whole event is a reference to Dragon Quest.

**Review by robflop — CHANGES_REQUESTED** [review]

**Comment by FabulousCupcake** [comment]
@stardustichi could you update (or not update) the PR with @robflop's suggestions so we can merge this?

**Comment by FabulousCupcake** [comment]
@stardustichi let me know if I can help with resolving the merge conflicts to get this merged 👀

File: src/data/trainer-title.csv (hunk 7 above)
Comment (FabulousCupcake) [reply src/data/trainer-title.csv]: Resolved via 6f87e9f; please update your review status

**Comment by stardustichi** [comment]
> @stardustichi let me know if I can help with resolving the merge conflicts to get this merged 👀

i think i resolved it, let me know if anything wrong lol, sorry for being late as always

**Review by robflop — APPROVED** [review]

**Review by FabulousCupcake — APPROVED** [review]
lgtm
